src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -35,12 +38,29 @@
   --pf-c-text-input-group__utilities--c-button--PaddingRight: var(--pf-global--spacer--xs);
   --pf-c-text-input-group__utilities--c-button--PaddingLeft: var(--pf-global--spacer--xs);
 
+  // input disabled style
+  --pf-c-text-input-group--m-disabled--Color: var(--pf-global--disabled-color--100);
+  --pf-c-text-input-group--m-disabled--BackgroundColor: var(--pf-global--disabled-color--300);
+
   position: relative;
   display: flex;
+  color: var(--pf-c-text-input-group--Color, inherit);
+  background-color: var(--pf-c-text-input-group--BackgroundColor);
 
   &:hover {
     --pf-c-text-input-group__text--after--BorderBottomColor: var(--pf-c-text-input-group--hover__text--after--BorderBottomColor);
   }
+
+  &.pf-m-disabled {
+    --pf-c-text-input-group--Color: var(--pf-c-text-input-group--m-disabled--Color);
+    --pf-c-text-input-group__icon--Color: var(--pf-c-text-input-group--m-disabled--Color);
+    --pf-c-text-input-group--BackgroundColor: var(--pf-c-text-input-group--m-disabled--BackgroundColor);
+
+    pointer-events: none;
+
+    --pf-c-text-input-group__text--before--BorderWidth: 0;
+    --pf-c-text-input-group__text--after--BorderBottomWidth: 0;
```

Comment:
Would you mind moving these vars to the top of this block? We follow the format of

```
.selector {
  // vars
  --var: whatever;

  // properties
  property: whatever;

  // combined selectors
  &.pf-m-whatever[:not(.something)] {
    // stuff
  }

  // combinators, descendent selectors, etc
  .some-descendent {
    // stuff
  }
  ~ .some-sibling {
    // stuff
  }
}
```